src/dump.rs

```diff
@@ -103,8 +104,8 @@ pub struct DumpClass {
 
     pub members: Vec<DumpClassMember>,
 
-    #[serde(default = "Vec::new")]
-    pub tags: Vec<String>,
+    #[serde(default = "BTreeSet::new")]

```

Comment:
```suggestion
    #[serde(default)]
```

I didn't know you could do this when I wrote this code, but for types that implement `Default`, Serde can use that implementation!